src/main/java/ui/Command.java

```diff
@@ -0,0 +1,39 @@
+package ui;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public enum Command {
+
+    EXIT("bye"),
+    LIST("list"),
+    MARK("mark", "mark <task #>"),
+    UNMARK("unmark", "unmark <task #>"),
+    DELETE("delete", "delete <task #>"),
+    TODO("todo", "todo <description>"),
+    DEADLINE("deadline", "deadline <description> /by <deadline>"),
+    EVENT("event", "event <description> /from <start> /to <end>");
+
+    // All commands that create a new task
+    public static final Set<String> ADD_TASK_COMMANDS = new HashSet<>(
+        Arrays.asList(TODO.label, DEADLINE.label, EVENT.label));
```

Comment:
Follow the Java coding standard here, "Indentation for wrapped lines should be 8 spaces". Similar issue  below.